aeon/dj_pipeline/analysis/block_analysis.py

```diff
@@ -1496,6 +1496,46 @@ def make(self, key):
         self.insert1(entry)
 
 
+# ---- Foraging Bout Analysis ----
+
+@schema
+class BlockForaging(dj.Computed):
+    definition = """
+    -> BlockSubjectAnalysis
+    ---
+    bout_count: int  # number of foraging bouts in the block
+    """
+
+    class Bout(dj.Part):
+        definition = """
+        -> master
+        -> BlockAnalysis.Subject
+        bout_start: datetime(6)
+        ---
+        bout_end: datetime(6)
+        bout_duration: float  # (seconds)
```

Comment:
So I guess the docstring and comments in that function, and any code computing the durations should be updated as welll?

Reply:
Ah, that's because I saw the `bout_durations` being computed in the function but then not used at all. I thought it was meant to be added to the result dataframe.
I think it's convenient to add in the table, but we can omit and compute on the fly if needed

Reply:
Oh, I now see that you mentioned this in #427, sorry I missed this

Reply:
Updated